modules/mobile/spec/support/fixtures/claims_and_appeals.json

```diff
@@ -5,7 +5,7 @@
     "data": {},
     "created_at": "2020-04-05T18:35:16.414Z",
     "updated_at": "2020-04-05T18:35:16.414Z",
-    "user_uuid": "3097e489-ad75-5746-ab1a-e0aabc1b426a",
+    "user_uuid": "885b29b2-87bf-5791-b7ca-2bb86cf3d6f5",
```

Comment:
from bullet 4 in the definition, I replaced all references of the old test data UUID with the new one